src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -125,7 +125,7 @@ public static FormulaValue CountRows(IRContext irContext, TableValue[] args)
             var arg0 = args[0];
 
             // Streaming 
-            var count = arg0.Rows.Count();
+            var count = arg0.Count();
```

Comment:
CountRows should return an error if there are error records in the table. I know that this makes the operation O(n) instead of O(1), but we need that for consistency.

Reply:
ack - but this is existing code. So that means we're missing a test...